sklearn/ensemble/tests/test_forest.py

```diff
@@ -1498,14 +1498,16 @@ def test_n_features_deprecation(Estimator):
 
 
 # TODO: Remove in v1.2
-def test_mse_deprecated():
-    est1 = RandomForestRegressor(criterion="mse", random_state=0)
+@pytest.mark.parametrize("criterion", ["mse", "mae"])
+def test_criterion_deprecated(criterion):
```

Comment:
Rather than storing the dictionary to map between the names, we can have them in parametrize:

```suggestion
@pytest.mark.parametrize("old_criterion, new_criterion", [
	("mse", "squared_error"),
	("mae", "absolute_error"),
])
def test_criterion_deprecated(old_criterion, new_criterion):
```